webapp/pages/profile/_id/_slug.vue

```diff
@@ -326,6 +280,8 @@ export default {
       tabActive: 'post',
       filter,
       followedByCountStartValue: 0,
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T14:04:16Z" title="Tuesday, April 7th 2020, 4:04:16 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -310,12 +263,14 @@ export default {
     HcUpload,
     MasonryGrid,
     MasonryGridItem,
+    FollowList,
   },
   transition: {
     name: 'slide-up',
     mode: 'out-in',
   },
   data() {
+    console.log('component data()')
```


this can go?

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-09T21:17:09Z" title="Thursday, April 9th 2020, 11:17:09 pm +02:00">Apr 9, 2020</time>_

---


2c7ca87ac